test/test2.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
```

Comment:
A description of test2 and test would be nice to add

Reply:
All tests are currently encapsulated in folders together together with corresponding files.